src/app/card/card.component.html

```diff
@@ -4,7 +4,7 @@
 
   <div class="card-content {{ textClass }}">
 
-    <span class="card-title">
+    <span class="card-title" *ngIf="title !== undefined">
```

Comment:
use `*ngIf="title"` ... it will validate empty string, null and undefined

![image](https://user-images.githubusercontent.com/6987084/29389742-fcc236bc-82b9-11e7-9470-55f4933a97c1.png)